Automattic/jetpack
# VideoPress: enable uploading button when site supports paid feature

Fixes https://github.com/Automattic/jetpack/issues/26870

#### Changes proposed in this Pull Request:

* VideoPress: disable upoading button when site supports paid feature

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test with a site with a free plan ([JN](https://href.li/?https://jurassic.ninja/create/?jetpack-beta&branches.jetpack-videopress=master&wp-debug-log&nojetpack))
* Go to the VideoPress dashboard
* Connect by clicking on `Start for free` button
* Once connected, upload a video
* Confirm you cannot upload a new one because of the free plan
* Get the paid feature
* Confirm when the client goes back to the dashboard
* Confirm the uploading button is disabled

before | after
------|------
<img width="645" alt="image" src="https://user-images.githubusercontent.com/77539/196194356-87e247be-c7ec-40dd-9a7e-66b4a75cf560.png"> | <img width="757" alt="image" src="https://user-images.githubusercontent.com/77539/196194528-1d36caad-9f7e-4539-842f-82feceaef733.png">
